.github/workflows/ROMROM-CHANGELOG-UPDATE.yaml

```
@@ -374,7 +374,10 @@ jobs:
-          # Auto-merge 활성화 (브랜치가 업데이트되면 자동으로 merge)
-          gh pr merge $PR_NUMBER --merge --auto
-          echo "✅ Auto-merge 활성화 완료!"
-          echo "🎯 deploy 브랜치가 업데이트되면 자동으로 merge"
\ No newline at end of file
+          # 브랜치 업데이트
+          gh api --method PUT /repos/${{ github.repository }}/pulls/${PR_NUMBER}/update-branch
+          sleep 5
+
+          # 즉시 merge
+          gh pr merge $PR_NUMBER --merge
+          echo "✅ PR merge 완료!"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)